test/index.html

```diff
@@ -134,7 +149,13 @@
           operationName: parameters.operationName,
           onEditQuery: onEditQuery,
           onEditVariables: onEditVariables,
-          onEditOperationName: onEditOperationName
+          onEditOperationName: onEditOperationName,
+          plugins: [
+            {
+              title: 'Example',
+              plugin: <Plugin render={viewer} />,
```

Comment:
This is just a dummy `Plugin` component, but in theory one could create multiple different types of Plugin components for different types. This could even be it's own library of components. 
eg. one with CodeMirror contained in the component